packages/terra-status/package.json

```diff
@@ -39,6 +39,10 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
+    "terra-arrange": "^2.27.0",
+    "terra-doc-template": "^1.18.0",
+    "terra-icon": "^2.25.0",
+    "terra-visually-hidden-text": "^1.0.0",
```

Comment:
`terra-arrange`, `terra-doc-template` and `terra-icon` are defined twice. Lets remove the older dependency versioned references in favor of the latest.

Then lets move `"terra-visually-hidden-text": "^1.0.0",` to the end of the list.